tests/TestCase/Database/Log/LoggedQueryTest.php

```diff
@@ -111,6 +112,22 @@ public function testStringInterpolationSpecialChars()
         $this->assertEquals($expected, (string)$query);
     }
 
+    /**
+     * Tests that query placeholders are replaced when logged
+     *
+     * @return void
+     */
+    public function testBinaryInterpolation()
+    {
+        $query = new LoggedQuery();
+        $query->query = 'SELECT a FROM b where a = :p1';
+        $uuid = str_replace('-', '', Text::uuid());
```

Comment:
Why would a normal text uuid be considered binary data? Why isn't the hex string printable? Wouldn't want want a normal uuid to be printed?

Reply:
Binary16 here, see issue.

Reply:
My mistake, I misread the query code below is all. I didn't have a problem with the actual change :)